stories/index.tsx

```diff
@@ -22,6 +22,7 @@ import '../src/layout/section/columnedContentSection/story'
 import '../src/layout/section/illustratedSection/story'
 import '../src/layout/section/itemsSection/story'
 import '../src/layout/section/mediaSection/story'
+import '../src/layout/section/slideSection/SlideSection.story'
```

Comment:
ping @gabrielagabriel isn't the import already handled somewhere else?

Reply:
Oh, I forgot to clean that up after I learned about the automagic import. Thanks for spotting it.

Reply:
Yes! No need to import stories here, this file was deleted. Something went wrong on a merge.